test/cookbooks/test/attributes/default.rb

```diff
@@ -5,6 +5,8 @@
 
 default['java']['jdk_version'] = if platform_family?('debian') && node['platform_version'].to_f < 9
                                    '7'
+                                 elsif platform_family?('debian') && node['platform_version'].to_f >= 10
```

Comment:
Ubuntu is also in the `debian` `platform_family` so maybe add an elsif above this to check if the platform is `ubuntu`?


from https://docs.chef.io/ruby.html 👇 
```
case node['platform']
when 'debian', 'ubuntu'
  # do debian/ubuntu things
when 'redhat', 'centos', 'fedora'
  # do redhat/centos/fedora things
end
For example, using a case statement with the platform_family node attribute:

case node['platform_family']
when 'debian'
  # do things on debian-ish platforms (debian, ubuntu, linuxmint)
when 'rhel'
  # do things on RHEL platforms (redhat, centos, scientific, etc)
end
```

Reply:
Right, it's the same as in the attributes… I'm on it.